test/services/analyzer/handlebars/helpers/wrap-quotes.unit.test.js

```diff
@@ -1,4 +1,4 @@
-const wrapQuotes = require('../../../../../src/services/dumper/handlebars/helpers/wrap-quotes');
+const wrapQuotes = require('../../../../../src/utils/handlebars/helpers/wrap-quotes');
```

Comment:
the source file has moved, you should move the test file to match

Reply:
👍